proto/v1/setting_service.proto

```diff
@@ -216,6 +219,20 @@ message ExternalApprovalSetting {
   repeated Node nodes = 1;
 }
 
+message SchemaTemplateSetting {
```

Comment:
We may still need category information. However, we could add this later if we are not sure if this is a string or map[string]string.

Reply:
Right, leave a TODO here